### ISSUE TITLE
"High Performance Computing (HPC) Job Opportunities" group on LinkedIn

### ISSUE THREAD

**Issue by sebastian-philipp** [hpc-social/jobs#23]
Hi there, 

Stewart Smith is maintaining a LinkedIn group called "High Performance Computing (HPC) Job Opportunities" containing job postings I'm not seeing here. 

Is there a way to unify those and show the same postings? I'd like to avoid looking into random LinkedIn groups. 

Thank you!

https://www.linkedin.com/groups/9084162/

**Comment by vsoch** [comment]
Not that I'm aware of - as far as I understand it, their API is concerned with logging in (as a user) and then posting or sharing. https://developer.linkedin.com/product-catalog. It's also not a good candidate for scraping because of the design of the pages, and needing to be a logged in member of the group to see said jobs. 

If you find a means and want to provide secrets / automation in the CI, please open a PR.

**Comment by sebastian-philipp** [comment]
Yeah. Alternative would be to reach out to him and discuss things.
